### PR TITLE
Improve readme

### LINKED ISSUE #41: Support socket request like sockjs-client or socket.io  ?

### LINKED ISSUE #61: Describe installation process
README should contain steps to install and use package.
Ideally at the beginning right after small package description.

`yarn add swr` - will it work?
`yarn add zeit/swr` - maybe this one?

How do I know which package belongs to which author?
How to verify I install your package and not some fake one?

### PR DESCRIPTION
**Pull request by quietshu**
We see people confusing about `fetch`, package names and other use cases. This PR tries to improve the README based on the feedback.

Close #41, close #61.

### CODE REVIEW

**Comment by quietshu** [comment]
To preview the rendered README: https://gist.github.com/quietshu/1bff81fc89b1c04e06bc28a674157c13

File: README.md [hunk 1]
```diff
@@ -182,6 +202,60 @@ function App () {
 }
 ```
 
+### Subscription (e.g.: socket.io)
```
Comment (sergiodxa) [inline README.md]: Should this code be inside the examples folder?

**Comment by quietshu** [comment]
@sergiodxa for `useSWRPages`, I want to ship #52 and some performance fixes before adding it to README, will do in 1~2 days.

File: README.md (hunk 1 above)
Comment (quietshu) [reply README.md]: @sergiodxa Yeah, I think we can add this model to your localStorage example using the `storage` event. So we can sync the data without focusing on the other window.

Comment (quietshu) [reply README.md]: Setting up socket.io for an example might be a bit overkill IMO.

**Pull request merged**